lib/Mojo/mysql.pm

```diff
@@ -149,6 +156,19 @@ Data Source Name, defaults to C<dbi:mysql:dbname=test>.
 Maximum number of idle database handles to cache for future use, defaults to
 C<5>.
 
+=head2 migrations
+
+MySQL does not support DDL transactions. B<Therefore, migrations should be used with extreme caution. Backup your database. You've been warned.> 
```

Comment:
I think this warning should be added to the Mojo::mysql::Migration module as well.